Project.toml

```diff
@@ -56,8 +56,8 @@ LegendDataManagement = "0.2.1"
 LegendDataTypes = "0.1"
 LegendHDF5IO = "0.1"
 LegendTextIO = "0.1"
-LsqFit = "0.13, 0.14"
-MJDSigGen = "0.1"
+LsqFit = "0.13, 0.14, 0.15"
+MJDSigGen = "0.1, 0.2"
```

Comment:
Does it make sense to restrict to `MJDSigGen` version `0.2` and drop support for `0.1`?
The changes here might not work with `0.1`.

Reply:
Yes, it's fine to require MJDSigGen v0.2